docs/changelog.md

```diff
@@ -4,6 +4,43 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.8 - May 18, 2023
+
+### New features and improvements
+
+- Data recovery is now up to 6x faster depending on the number of available cores, as
+  [snapshot loading is distributed among several
+  threads](/memgraph/reference-guide/backup#snapshots).
+- Indexes can be created using multiple threads, thus speeding up the recovery
+  process. [#882](https://github.com/memgraph/memgraph/pull/882)
+  [#868](https://github.com/memgraph/memgraph/pull/868)
```

Comment:
This is in the wrong spot, memgraph#868 is the faster recovery part (the first item)

Reply:
Probably got wrongly merged or something

Reply:
yup! thnx